dnf5/commands/repoquery/repoquery.cpp

```diff
@@ -256,6 +256,16 @@ void RepoqueryCommand::set_argument_parser() {
         "Limit to unneeded installed packages (i.e. packages that were installed as "
         "dependencies but are no longer needed).",
         false);
+    extras = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this, "extras", '\0', "Limit to packages that are not present in any of of available repositories.", false);
+    upgrades = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this,
+        "upgrades",
+        '\0',
+        "Limit to packages that provide an upgrade for some already installed package.",
+        false);
+    recent = std::make_unique<libdnf::cli::session::BoolOption>(
+        *this, "recent", '\0', "Limit to only recently edited packages.", false);
```

Comment:
I know it's also in dnf4 docs, but I consider word `edited` a bit odd in this context. Wouldn't be `changed` or `added` a better fit?

Reply:
`added` seems a bit misleading to me but `changed` is better, thanks. :+1: